packages/ia-components/sandbox/details/CrawlConfig.js

```diff
@@ -0,0 +1,145 @@
+import ReactDOM from "react-dom";
+
+const debug = require('debug')('dweb-archive:CrawlConfig');
+const canonicaljson = require('@stratumn/canonicaljson');
```

Comment:
are these able to be imports?  or do they need require?

Reply:
debug definately needs require (it has to be loaded before the debug statement) 
canonicaljson probably doesn't ... I'll make that change.

Reply:
Done